main.py

```diff
@@ -26,16 +26,23 @@ def create_menu():
     menu.add_option('4', 'Show Read Books', show_read_books)
     menu.add_option('5', 'Show All Books', show_all_books)
     menu.add_option('6', 'Change Book Read Status', change_read)
+    menu.add_option('7', 'Delete Book', delete_book)
     menu.add_option('Q', 'Quit', quit_program)
 
     return menu
 
 
 def add_book():
     new_book = ui.get_book_info()
-    new_book.save()
+    new_book.save()\
+
+def delete_book():
+    try:
+        which_book = ui.get_book_id()
```

Comment:
A better variable name would be which_book_id or book_id or id_to_delete. This variable stores an id, not a book